evaluate/metrics/assessment_test.go

```diff
@@ -134,24 +134,27 @@ func TestAssessmentString(t *testing.T) {
 	}
 
 	validate(t, &testCase{
-		Name: "Initial Metrics",
+		Name: "Empty Metrics",
 
 		Assessment: NewAssessments(),
 
-		ExpectedString: "files-executed=0, files-problems=0, coverage-statement=0, response-no-excess=0",
+		ExpectedString: "files-executed=0, files-problems=0, coverage-statement=0, response-no-error=0, response-not-empty=0, response-with-code=0, response-no-excess=0",
 	})
 
 	validate(t, &testCase{
-		Name: "Empty Metrics",
+		Name: "Value Metrics",
 
 		Assessment: Assessments{
 			AssessmentKeyCoverageStatement: 1,
 			AssessmentKeyFilesExecuted:     2,
 			AssessmentKeyFilesProblems:     3,
-			AssessmentKeyResponseNoExcess:  4,
+			AssessmentKeyResponseNoError:   4,
+			AssessmentKeyResponseNoExcess:  5,
+			AssessmentKeyResponseNotEmpty:  6,
+			AssessmentKeyResponseWithCode:  7,
 		},
 
-		ExpectedString: "files-executed=2, files-problems=3, coverage-statement=1, response-no-excess=4",
+		ExpectedString: "files-executed=2, files-problems=3, coverage-statement=1, response-no-error=4, response-not-empty=6, response-with-code=7, response-no-excess=5",
```

Comment:
Wondering. Headers right now are not deterministic. let's sort them by alphabet right away?

Reply:
They are. The ordering is always the same as how the assessment keys are registered.

Reply:
Not if you are registering keys from other packages. That is IMO where we are heading: common task-packages that define their keys.